### DISCUSSION TITLE
Balance non replicated queues

### DISCUSSION THREAD

**Discussion by MonicaMagoniCom** [rabbitmq/rabbitmq-server#6970] [Questions]
Hi,

I have enabled classic queues without replication (in our scenario is not needed to have persistency) but I have encountered the following problem: 
when a replica of the cluster is restarted (for any reason), then all the queues got concentrated on all the other replicas. Is not possible to rebalance the queues?

**Comment by michaelklishin** [comment]
```
rabbitmq-queues help rebalance
```

**Reply by MonicaMagoniCom** [reply]
As far as I know, this does not work for non replicated queues.
